FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -429,13 +429,16 @@ def getInstallations( self, argss ):
         else:
           installedBy = installation[ 'InstalledBy' ]
 
+        if not installation[ 'UnInstalledBy' ]:
+          uninstalledBy = ''
+        else:
+          uninstalledBy = installation[ 'UnInstalledBy' ]
```

Comment:
Then it should rather be:
uninstalledBy = installation.get('UnInstalledBy', '')
if uninstalledBy is None:
  uninstalledBy = ''